### DISCUSSION TITLE
Mosquito trouble in Dynamo campaign

### DISCUSSION THREAD

**Discussion by Nocke216** [Khopa/dcs_liberation#2198] [Q&A] [answered]
Hi there,

I am completely new to this - so please bear with me :)

The only DCS module I own is the Mosquito, so I started as my first campaign with the Dynamo scenario, set time and factions to 1944.
I can edit flights containing other planes, but whenever I try to edit a flight with Mosquitos, I get this:

Traceback (most recent call last):
  File "widgets\ato.py", line 119, in on_double_click
  File "widgets\ato.py", line 124, in edit_flight
  File "dialogs.py", line 57, in open_edit_flight_dialog
  File "windows\mission\QEditFlightDialog.py", line 33, in __init__
  File "windows\mission\flight\QFlightPlanner.py", line 20, in __init__
  File "windows\mission\flight\payload\QFlightPayloadTab.py", line 39, in __init__
  File "windows\mission\flight\payload\QFlightPayloadTab.py", line 13, in __init__
  File "gen\flights\loadouts.py", line 120, in iter_for
  File "gen\flights\loadouts.py", line 120, in <dictcomp>
  File "game\data\weapons.py", line 78, in with_clsid
KeyError: ''

Is it maybe too early to use the Mossie? No loadouts defined? Can I do something there?

Thank you very much for any hint!

**Comment by Starfire13** [comment, marked as answer]
Remove the rockets from your loadouts.

**Comment by Nocke216** [comment]
Thank you ... but where do I do that, the problem being that I can't get to
editing the flight?

Starfire13 ***@***.***> schrieb am So., 1. Mai 2022, 05:15:

**Reply by Starfire13** [reply]
In the DCS mission editor. You have to remove rockets from all your Mosquito loadouts, not just those used by Liberation.

**Reply by Nocke216** [reply]
I am new to DCS, haven't ever seen that editor yet. Sigh ....
Does it mean in the editor I can define configurations which then will be used all over the game, in any mission?
In all mission editors from other games I always created or opened a mission to work on.
How to proceed here, if in the first place I am not able to create a mission to edit?
And thx again!

**Reply by Nocke216** [reply]
So I went into the FMB, placed a Mosquito on a map, went through all the mission tasks, removed rockets from all payload lines where I found them, exporting each and every one I changed.
Hoping I did not mess up my installation ...
But it did not help, the error remains.

**Reply by Starfire13** [reply]
Sorry I'm not understanding what you're doing. What are you exporting and why? There is no exporting involved in the process. Can you get on discord and post a few screenshots of what you're doing?

**Comment by Nocke216** [comment]
Would be fantastic if we could meet on discord for that, thank you!
Which discord?  My nick on discord is a little different: 216th_Nocke#9256

Starfire13 ***@***.***> schrieb am Mo., 2. Mai 2022, 02:57:

**Reply by Starfire13** [reply]
The Liberation Discord. Link is on the Github page. There is a help channel.
